### PR TITLE
Remove port from url

### PR DESCRIPTION
**Pull request by joinsito**
Hi,

I've found an issue when trying to access my application through valet specifying the port in the address, valet is not able to find the right folder in this case.

My motivation for specifying the port is that I'm accessing my application through a reverse proxy in a remote server listening on a port different from 443 or 80.

 I've fixed it by removing port from $_SERVER['HTTP_HOST'].

### CODE REVIEW

File: server.php [hunk 1]
```diff
@@ -65,8 +65,8 @@ function valet_default_site_path($config)
 
 $siteName = basename(
     // Filter host to support wildcard dns feature
-    valet_support_wildcard_dns($_SERVER['HTTP_HOST']),
-    '.'.$valetConfig['tld']
+    valet_support_wildcard_dns(explode(':', $_SERVER['HTTP_HOST'])[0]),
+    '.'.$valetConfig['domain']
```
Comment (drbyte) [inline server.php]: Hi, `domain` here should be changed to `tld`

Comment (joinsito) [reply server.php]: Noted, thanks.

**Comment by drbyte** [comment]
@joinsito What are your thoughts on whether it's best to remove the port before passing it into the `valet_support_wildcard_dns()` function, vs doing it inside the `valet_support_wildcard_dns()` function just before it returns the parsed domain, as in #630 ?

**Comment by joinsito** [comment]
@drbyte, makes sense to do it inside the function as in #630. Happy to see It merged!

**Comment by drbyte** [comment]
Great!  Thanks for your contribution @joinsito !

**Pull request closed without merging**